tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -381,8 +391,11 @@ build_rootfs_distro()
 
 		# setup to install rust here
 		generate_dockerfile "${distro_config_dir}"
-		"$container_engine" build  \
+		[ "${CROSS_BUILD}" == "true" ] && dockerfile_name="Dockerfile.cross" || dockerfile_name="Dockerfile"
+		"$container_engine" build \
 			${engine_build_args} \
+			-f "${distro_config_dir}"/"${dockerfile_name}" \
+			--build-arg ARCH="${ARCH}" \
```

Comment:
Could you add the following:

```
--env CROSS_BUILD="${CROSS_BUILD}" \
```

It will be used for adding a toolchain target (e.g. `s390x` or `ppc64le`) during the rust installation (please see https://github.com/kata-containers/tests/pull/5618)

Reply:
OK